python/pyproject.toml

```diff
@@ -45,6 +45,11 @@ requires = [
 build-backend = "mesonpy"
 
 [tool.meson-python.args]
+# Consistent version for zstd, rather than attempting to delocate whatever
+# version was on the build host.
+# TODO: We need to override this somehow on conda-forge because we actually
+# do want the host fallback.
+setup = ['--force-fallback-for=zstd']
```

Comment:
Giving this another review I think you should actually move this to the `meson.build` configuration on L28, right below the setting we have for `default_library=static`; that way, all of the configuration is contained in one place, rather than split between pyproject.toml and the meson config

Reply:
Looks like it needs to be `force_fallback_for` (underscores instead of hyphens)

See also https://mesonbuild.com/Builtin-options.html#core-options